src/components/forms/PatientForm.tsx

```diff
@@ -23,7 +23,7 @@ const PatientForm: FunctionComponent<PatientFormProps> = ({ patient, isEditForm
   const { t } = useTranslationHook()
   const navigate = useNavigate()
 
-  const birthdayDateValue: Date | null = patient.birthday !== '' ? new Date(patient.birthday) : null
+  const birthdayDateValue: Date | null = patient.birthday !== null && patient.birthday !== undefined && patient.birthday !== '' ? new Date(patient.birthday) : null
```

Comment:
Would this work?
```
const birthdayDateValue: Date | null = patient.birthday ? new Date(patient.birthday) : null;
```

Reply:
no I have to check `Unexpected nullable string value in conditional.`

Reply:
OK linter wants it explicit which makes sense but it's painful in this case